scripts/ci/libraries/_initialization.sh

```diff
@@ -511,7 +511,7 @@ function initialization::initialize_provider_package_building() {
 # Determine versions of kubernetes cluster and tools used
 function initialization::initialize_kubernetes_variables() {
     # Currently supported versions of Kubernetes
-    CURRENT_KUBERNETES_VERSIONS+=("v1.20.2" "v1.19.7" "v1.18.15")
+    CURRENT_KUBERNETES_VERSIONS+=("v1.21.1" "v1.20.2")
```

Comment:
@potiuk, should these be flipped so the default is the oldest?

Reply:
Yes. That's indeed better idea.

Reply:
done